.gitignore

```diff
@@ -8,3 +8,4 @@ figures
 /TODO.txt
 output
 mrremind.Rproj
+mrremind_rob.Rproj
```

Comment:
Instead of adding your own file name, just change the line above to `*.Rproj`

Reply:
thanks, good to know!